src/middlewared/middlewared/alert/source/cores.py

```diff
@@ -16,18 +22,37 @@ class CoreFilesArePresentAlertClass(AlertClass):
 
 
 class CoreFilesArePresentAlertSource(AlertSource):
+    dumps = {}
     schedule = IntervalSchedule(timedelta(minutes=5))
 
     products = ("SCALE",)
 
     async def check(self):
         corefiles = []
-        for coredump in await self.middleware.call("system.coredumps"):
-            if coredump["corefile"] == "present":
-                if coredump["exe"] == "/usr/sbin/syslog-ng":
+        for coredump in filter(lambda c: c["corefile"] == "present", await self.middleware.call("system.coredumps")):
+            if coredump["exe"] == "/usr/sbin/syslog-ng":
+                continue
+
+            if coredump["pid"] not in self.dumps:
+                cp = await run(
+                    "coredumpctl", "info", str(coredump["pid"]), check=False, encoding="utf-8", errors="ignore"
+                )
+                if cp.returncode:
+                    self.middleware.logger.debug(
```

Comment:
This will log the same message every minute. We should probably cache the error state too to avoid calling `coredumpctl info` for failing pid every minute.